plugins/pom-analyzer/src/main/java/eu/f4sten/pomanalyzer/Main.java

```diff
@@ -123,9 +123,10 @@ private void consumeWithTimeout(MavenId id, Lane lane) {
         } catch (InterruptedException e) {
             Thread.currentThread().interrupt();
         } catch (ExecutionException e) {
-            // this should not happen
             var cause = e.getCause();
-            if (cause instanceof RuntimeException) {
+            if (cause instanceof Error) {
```

Comment:
Maybe it's a good idea to catch the specific exception that is thrown, `UnrecoverableError`, rather than `Error`.

Reply:
The logic should catch all errors, not just our `UnrecoverableErrors` that we use to wrap all kinds of things to make the app crash...